packages/menu/src/MenuItem.ts

```diff
@@ -369,6 +379,7 @@ export class MenuItem extends LikeAnchor(
         import('@spectrum-web-components/popover/sp-popover.js');
         return html`
             <sp-overlay
+                receives-focus="false"
```

Comment:
We don't want focus automatically set or the focus indicator would be one step ahead of mouse hover.